lib/phoenix/endpoint/supervisor.ex

```diff
@@ -84,9 +84,9 @@ defmodule Phoenix.Endpoint.Supervisor do
     children =
       config_children(mod, secret_conf, default_conf) ++
         pubsub_children(mod, conf) ++
-        socket_children(mod, :child_spec) ++
+        socket_children(otp_app, mod, :child_spec) ++
```

Comment:
Instead of passing the `otp_app`, please pass the `conf` instead. This way, you don't have to read it again.